src/fabric/RNGestureHandlerRootViewNativeComponent.ts

```diff
@@ -0,0 +1,12 @@
+// @ts-ignore TS being TS
```

Comment:
Let's add a note why we need to ignore these types with a TODO not to forget about it in the future.

Reply:
Done in [1151519](https://github.com/software-mansion/react-native-gesture-handler/pull/1952/commits/1151519599c4b31a3d0ed0eb22c888aa6b72ed53).